### DISCUSSION TITLE
[Enhancement] 暗黑模式下修改某个组件scss变量值的问题

### DISCUSSION THREAD

**Discussion by slient7** [element-plus/element-plus#16696] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `All`

## 需要能够在暗黑模式下，修改所有组件的默认scss变量值
```scss
@forward "element-plus/theme-chalk/src/dark/var.scss" with (
  $table: (
    'header-text-color': #FFFFFF,
    'header-bg-color': #1D3A6A,
    'tr-bg-color': #162B5A,
    'row-hover-bg-color': #132A50,
  )
);
```
如上，修改暗黑模式下的table组件时会提示，[plugin:vite:css] [sass] This variable was not declared with !default in the @used module.
只能在@forward "element-plus/theme-chalk/src/common/var.scss"，里面修改才能生效。
这样会有个问题，切换回浅色模式后，table组件显示的还是深色的样式。

### Additional comments

_(empty)_

![1714299790556](https://github.com/element-plus/element-plus/assets/135320692/4cd9fc69-45f4-4b07-8026-1a3cae0106e7)

**Comment by maplelost** [comment]
没有人干这件事唉,我们看看怎么干?我现在也需要这功能

**Reply by maplelost** [reply]
oh我找到了 https://github.com/element-plus/element-plus/issues/9076
